.env.docker

```diff
@@ -0,0 +1,24 @@
+# Application
+NODE_ENV=production
+SENTRY_DNS=
+
+# Database
+DB_HOST=mongo:27017
+DB_USERNAME=algorand
+DB_PASSWORD=algorand
+DB_NAME=fido
+DB_ATLAS=false
+
+# Events
+REDIS_HOST=redis
+REDIS_PORT=6379
+REDIS_USERNAME=default
+REDIS_PASSWORD=
+
+# FIDO2
+RP_NAME=Algorand Foundation FIDO2 Server
+HOSTNAME=catfish-pro-wolf.ngrok-free.app
```

Comment:
Did you mean to check in this exact .env.docker file with your ngrok static domain?

Reply:
We could make it a template or remove it entirely. It's not used in the JS library directly and shouldn't effect this release.

Reply:
I don't think there is any value in optimizing developer resources in a canary release